packages/react-api/tsconfig.build.json

```diff
@@ -3,7 +3,10 @@
   "compilerOptions": {
     "baseUrl": "..",
     "outDir": "./build",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "lib": [
+      "es2019"
```

Comment:
Please remove this - this is non-standard for the full project, hence problematic. (We aim for consistency as much as possible, not only within this repo, but also across all in polkadot-js, hence the shared configs)